account/image.go

```diff
@@ -0,0 +1,68 @@
+// Package account stores and tracks accounts within a Chain Core.
+package account
+
+import (
+	"encoding/json"
+)
+
+type AccountSlice struct {
+	Account       *Account
+	ContractIndex uint64
+}
+
+type AccountImage struct {
```

Comment:
`AccountImage` maybe not a good naming. Package name has 'account' info, so maybe just `Image` is enough.